src/web/components/Core/StatusPopup.scss

```diff
@@ -2,64 +2,50 @@
   position: fixed;
   top: 23px;
   right: 31px;
-  padding: 24px;
-  border-radius: 5px;
+  width: 345px;
+  padding: 20px;
+  border-radius: 10px;
   z-index: 1000;
   background: var(--theme-background-content);
   box-shadow: 0px 6px 12px var(--theme-box-shadow);
-  border-radius: 4px;
-  border-left: 8px solid;
-
-  &.Success {
-    border-color: var(--theme-success);
-  }
-
-  &.Error {
-    border-color: var(--theme-error);
-  }
-
-  &.Info {
-    border-color: var(--theme-action);
-  }
-}
-
-.popup-close-button {
-  position: absolute;
-  top: 0px;
-  right: 0px;
-  background: none;
-  border: none;
-}
-
-.popup-close-icon {
-  width: 7.5px;
+  border-radius: 10px;
 }
 
 .status-popup-title-container {
   margin-bottom: 4px;
-  &.Success {
-    color: var(--theme-success);
-  }
+  display: flex;
+  justify-content: space-between;
 
-  &.Error {
-    color: var(--theme-error);
+  svg {
+    font-size: 1rem;
   }
 
-  &.Info {
-    color: var(--theme-action);
+  .status-popup-title {
+    margin-left: 8px;
+    font-weight: 700;
+    font-size: 1rem;
+    line-height: 1rem;
+
+    .status-popup-title-left {
```

Comment:
I don't think this css is actually being applied: the DOM has `status-popup-title` inside `status-popup-title-left` rather than the other way around. Perhaps we could get rid of this className?

Reply:
good call.  i got carried away with keeping things symmetric.